parsigs/parse_sig_api.py

```diff
@@ -127,9 +134,9 @@ def _pre_process(sig):
     for word in words:
         if word == 'tab':
             word = word.replace('tab', 'tablet')
-            output_words.append(word)
-        else:
-            output_words.append(word)
+        elif word == 'tabs':
+            word = word.replace('tabs', 'tablet')
+        output_words.append(word)
```

Comment:
you changed the logic here - and appending the word twice
there need to be an `else` before line 139